sam-app2/HelloWorldFunction/src/main/resources/collector.yaml

```diff
@@ -0,0 +1,28 @@
+receivers:
```

Comment:
This file could also sit in an S3 location, only readable within the org (or within OUs) - therefore could contain the API key/secret and only accessible to IAM roles in the relative OU?